src/dodal/devices/watsonmarlow323.py

```diff
@@ -0,0 +1,53 @@
+from enum import Enum
+
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_rw
+
+class WatsonMarlow323Enable(str, Enum):
+     DISABLED = "Disabled"
+     ENABLED = "Enabled"
+
+class WatsonMarlow323Direction(str, Enum):
+     CW = "CW"
+     CCW = "CCW"
+
+class WatsonMarlow323State(str, Enum):
+     STOPPED = "STOP"
+     STARTED = "START"
+
+class WatsonMarlow323(StandardReadable):
+     """Watson Marlow 323 Pump device"""
+     def __init__(self, prefix: str, name: str= "") -> None:
+          
+          self.enabled = epics_signal_r(WatsonMarlow323Enable, prefix + "DISABLE")
```

Comment:
I haven't read the manual, but is this PV really read-only?

Reply:
Checking this its not read only so I've made it rw in 417aaa12897e1aad61c73db30d4f3d6e00974e29.